datahub-web-react/src/graphql/role.graphql

```diff
@@ -0,0 +1,50 @@
+query listRoles($input: ListRolesInput!) {
+    listRoles(input: $input) {
+        start
+        count
+        total
+        roles {
+            urn
+            type
+            name
+            description
+            relationships(input: { types: ["IsMemberOfRole", "IsAssociatedWithRole"], direction: INCOMING, start: 0 }) {
```

Comment:
Why 2 roles? Can we split into multiple fields? 

users: 

policies: 

set a limit here and then it will make this query a bit cheaper.

Reply:
users: relationships 

policies: relationships

Reply:
Split! Good suggestion